tests/suites/test_suite_pkwrite.function

```diff
@@ -28,12 +28,22 @@ void pk_write_pubkey_check( char * key_file )
     ret = mbedtls_pk_write_pubkey_pem( &key, buf, sizeof( buf ));
     TEST_ASSERT( ret == 0 );
 
+    pem_len = strlen( (char *) buf );
+
+    // check that the rest of the buffer remains clear
+    for( buf_index = pem_len ; buf_index < 5000; ++buf_index )
+    {
+        TEST_ASSERT( buf[buf_index] == 0 );
+    }
+
+    TEST_ASSERT((5000 - pem_len) > 2);
```

Comment:
Style:
```suggestion
    TEST_ASSERT( (5000 - pem_len) > 2 );
```

Reply:
Whoops, this wasn't meant to be here, this was just me testing that at least in one of tests the buffers was bigger than the data written to it. Removed